<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>582</width>
    <height>307</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_address">
          <property name="text">
           <string>address:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="addressEdit">
          <property name="text">
           <string>127.0.0.1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_port">
          <property name="text">
           <string>Port:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="portEdit">
          <property name="text">
           <string>6000</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="connect2ServerButton">
          <property name="text">
           <string>Connect</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QPushButton" name="axis_1_fw">
            <property name="text">
             <string>Axis 1 +</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="axis_2_fw">
            <property name="text">
             <string>Axis 2 +</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="axis_3_fw">
            <property name="text">
             <string>Axis 3 +</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="axis_4_fw">
            <property name="text">
             <string>Axis 4 +</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="axis_5_fw">
            <property name="text">
             <string>Axis 5 +</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="axis_6_fw">
            <property name="text">
             <string>Axis 6 +</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="axis_7_fw">
            <property name="text">
             <string>Axis 7 +</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QLCDNumber" name="lcdNum_axis_1">
            <property name="smallDecimalPoint">
             <bool>false</bool>
            </property>
            <property name="digitCount">
             <number>7</number>
            </property>
            <property name="segmentStyle">
             <enum>QLCDNumber::Flat</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLCDNumber" name="lcdNum_axis_2">
            <property name="digitCount">
             <number>7</number>
            </property>
            <property name="segmentStyle">
             <enum>QLCDNumber::Flat</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLCDNumber" name="lcdNum_axis_3">
            <property name="digitCount">
             <number>7</number>
            </property>
            <property name="segmentStyle">
             <enum>QLCDNumber::Flat</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLCDNumber" name="lcdNum_axis_4">
            <property name="digitCount">
             <number>7</number>
            </property>
            <property name="segmentStyle">
             <enum>QLCDNumber::Flat</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLCDNumber" name="lcdNum_axis_5">
            <property name="digitCount">
             <number>7</number>
            </property>
            <property name="segmentStyle">
             <enum>QLCDNumber::Flat</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLCDNumber" name="lcdNum_axis_6">
            <property name="digitCount">
             <number>7</number>
            </property>
            <property name="segmentStyle">
             <enum>QLCDNumber::Flat</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLCDNumber" name="lcdNum_axis_7">
            <property name="digitCount">
             <number>7</number>
            </property>
            <property name="segmentStyle">
             <enum>QLCDNumber::Flat</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QPushButton" name="axis_1_bw">
            <property name="text">
             <string>Axis 1 -</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="axis_2_bw">
            <property name="text">
             <string>Axis 2 -</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="axis_3_bw">
            <property name="text">
             <string>Axis 3 -</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="axis_4_bw">
            <property name="text">
             <string>Axis 4 -</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="axis_5_bw">
            <property name="text">
             <string>Axis 5 -</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="axis_6_bw">
            <property name="text">
             <string>Axis 6 -</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="axis_7_bw">
            <property name="text">
             <string>Axis 7 -</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>28</width>
            <height>210</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QPushButton" name="end_x_fw">
            <property name="text">
             <string>End X +</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="end_y_fw">
            <property name="text">
             <string>End Y +</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="end_z_fw">
            <property name="text">
             <string>End Z +</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="end_rx_fw">
            <property name="text">
             <string>End RX +</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="end_ry_fw">
            <property name="text">
             <string>End RY +</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="end_rz_fw">
            <property name="text">
             <string>End RZ +</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QLCDNumber" name="lcdNum_end_x">
            <property name="digitCount">
             <number>7</number>
            </property>
            <property name="segmentStyle">
             <enum>QLCDNumber::Flat</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLCDNumber" name="lcdNum_end_y">
            <property name="digitCount">
             <number>7</number>
            </property>
            <property name="segmentStyle">
             <enum>QLCDNumber::Flat</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLCDNumber" name="lcdNum_end_z">
            <property name="digitCount">
             <number>7</number>
            </property>
            <property name="segmentStyle">
             <enum>QLCDNumber::Flat</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLCDNumber" name="lcdNum_end_rx">
            <property name="digitCount">
             <number>7</number>
            </property>
            <property name="segmentStyle">
             <enum>QLCDNumber::Flat</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLCDNumber" name="lcdNum_end_ry">
            <property name="digitCount">
             <number>7</number>
            </property>
            <property name="segmentStyle">
             <enum>QLCDNumber::Flat</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLCDNumber" name="lcdNum_end_rz">
            <property name="digitCount">
             <number>7</number>
            </property>
            <property name="segmentStyle">
             <enum>QLCDNumber::Flat</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QPushButton" name="end_x_bw">
            <property name="text">
             <string>End X -</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="end_y_bw">
            <property name="text">
             <string>End Y -</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="end_z_bw">
            <property name="text">
             <string>End Z -</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="end_rx_bw">
            <property name="text">
             <string>End RX -</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="end_ry_bw">
            <property name="text">
             <string>End RY -</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="end_rz_bw">
            <property name="text">
             <string>End RZ -</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
